CodeGen/Program.cs

```diff
@@ -43,7 +43,8 @@ internal class Program
         /// <param name="verbose">Verbose output? Defaults to false.</param>
         /// <param name="repositoryRoot">The repository root directory, defaults to searching parent directories for UnitsNet.sln.</param>
         /// <param name="skipWrc">Skip generate UnitsNet.WindowsRuntimeComponent? Defaults to false.</param>
-        private static int Main(bool verbose = false, DirectoryInfo repositoryRoot = null, bool skipWrc = false)
+        /// <param name="skipNanoFramework">Skip generate nanoFrmaework Units</param>
```

Comment:
Typo.
Maybe: `Skip generate UnitsNet.nanoFramework quantities? Defaults to false.`